{"uuid": "37f95b23-0aa2-4d74-b5be-3827d9eb35e3", "counter": 5, "stdout": "\u001b[0;31mfatal: [192.168.10.5]: FAILED! => {\"changed\": false, \"msg\": \"command timeout triggered, timeout value is 30 secs.\\nSee the timeout setting options in the Network Debug and Troubleshooting Guide.\"}\u001b[0m", "start_line": 4, "end_line": 5, "runner_ident": "555a3c18-1d1d-45f8-936b-e93653673bef", "event": "runner_on_failed", "pid": 474864, "created": "2025-04-15T02:06:57.926456+00:00", "parent_uuid": "000c2948-06b9-aea7-5ae4-000000000004", "event_data": {"playbook": "ping.yml", "playbook_uuid": "b1a35a2b-416a-4096-a822-f798777ff697", "play": "Connectivity Test", "play_uuid": "000c2948-06b9-aea7-5ae4-000000000002", "play_pattern": "all", "task": "Ping a remote device from the UI", "task_uuid": "000c2948-06b9-aea7-5ae4-000000000004", "task_action": "ios_command", "resolved_action": "cisco.ios.ios_command", "task_args": "", "task_path": "/home/kali/Desktop/fyp/networkproject/networkmanagement/ansible/ping.yml:5", "host": "192.168.10.5", "remote_addr": "192.168.10.5", "res": {"msg": "command timeout triggered, timeout value is 30 secs.\nSee the timeout setting options in the Network Debug and Troubleshooting Guide.", "invocation": {"module_args": {"commands": ["ping "], "match": "all", "retries": 9, "interval": 1, "wait_for": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-04-15T02:06:26.008634+00:00", "end": "2025-04-15T02:06:57.926266+00:00", "duration": 31.917632, "ignore_errors": null, "event_loop": null, "uuid": "37f95b23-0aa2-4d74-b5be-3827d9eb35e3"}}